### DISCUSSION TITLE
[Design/Antagonist] Idea for Station Swarmers rework

### DISCUSSION THREAD

**Discussion by Contrabang** [ParadiseSS13/Paradise#18701] [Documents]
This is an alternative to the TG Swarmers idea, which to me, doesn't seems like it worked very well considering TG removed it themselves. This is probably one of the either worst, best, or funniest ideas I've ever had. This is more of a general idea than anything else and more for gauging interest of the design and balance teams.

## Current Issues

Swarmers...
- Are annoying to fight because they are ranged
- Are somewhat boring to play as because all you do is click all the things in the room then leave
- Don't actually SWARM
- Non-lethally forcing to call the shuttle because everything is gone
- Only weakness is EMPs
- Can leave any time they want by ventcrawling and can always replicate back to their natural numbers

## Proposed Solution 1
This would not affect lavaland swarmers, those are their own different problem in this scenario

Remove a majority of their abilities, including to ventcrawling, ranged disabler ability, melee shock ability, and ability to dismantle and deconstruct things.

Keep their hivemind ability, light ability, trap and barrier abilities, ability to slowly self repair, and ability to make another swarmer, and would still not be allowed to breach the station.

Then the key piece, give them a somewhat weak (5-10 damage) but fast (0.1 to 0.3 toolspeed) welder. This would allow them to reproduce much more quickly by welding down walls, but they will have trouble consuming objects that are not instantly weldable. It also makes them actually swarm, a pretty good trade imo for not being able to ventcrawl. This would also implement a new counter to swarmers beyond EMPs by building reinforced walls (which they wouldn't be able to deconstruct, but maybe they could be given a slow ability to cut through the wall in the course of 15 - 30 seconds?). This would also make them fun to fight because theyre dangerous and will try to kill you with their welder rather than just teleport you away (this would require changing their laws too). This would put them more on a blob/terror spider level of lethality where they are all in the same place (because they cant ventcrawl) and would require the entire crew together to remove them (which can actually be done because they cant ventcrawl away).

This would also introduce some interesting interactions like swarmer being able to weld doors shut and it being harder to break into areas, making them swarm all in one place. This would probably also require for swarmers to be moved to a Major midround because of their dangerous status and not spawning from the gateway because they wouldnt be able to ventcrawl out. It would probably be a good idea too for there to be some way for crew to get rid of swarmer shells, maybe just decreasing their durability and being easier to break would be good.

If you want to have an idea of what this might be like, I recommend getting an alien welder on a test server and just going ham, Its pretty fun.

## Proposed Solution 2
Keep most of their abilities, but remove their ranged disabler ability and make them melee stamina damage only and increase their speed along with removing their ventcrawling.

This stun melee could be expanded probably in some kind of way to be more interesting, but I think it is less promising than proposed solution one.

**Comment by PacifistDalek** [comment]
I like this, i feel like that option number 1 really keeps a lot of what makes swarmers appealing but removes most of the frustration. As someone whose favourite antagonist is blob, i like the blob-like elements of them swarming together

**Comment by AffectedArc07** [comment]
I still stand firmly on the removal train. My opinion is not the only opinion in the design team, but its what I stand by.
